client/src/components/ProfileSelection/ProfileSelection.tsx

```diff
@@ -0,0 +1,25 @@
+import useStyles from './useStyles';
+import Grid from '@material-ui/core/Grid';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+
+const ChatSideBanner = (props: { name: string; link: string; selected: boolean }): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <Grid container>
+      <Grid item xs>
+        <Button
+          component={Link}
```

Comment:
You can look into using react-router's `NavLink`, it has a prop called `activeClassName` which can be used rather than passing down a 'selected' prop to check whether the link has been selected (because currently you've hardcoded a single link as 'true', I presume you will be using a state to keep track of this - which is not necessary) 😄 

Ref: https://reactrouter.com/web/api/NavLink

Reply:
Using NavLink instead of link